client/js/components/user/orgaDataEntry/EmailNotificationSettings.vue

```diff
@@ -118,7 +118,7 @@ export default {
 
   computed: {
     hasNotificationSectionPermission () {
-      return this.willReceiveNewStatementNotification && hasPermission('feature_notification_statement_new') ||
+      return (this.willReceiveNewStatementNotification && hasPermission('feature_notification_statement_new')) ||
```

Comment:
Do we use this for better readability?

Reply:
Yes, eslint was complaining that we're mixing `&&` and `||` and that we should add parentheses so it's easer to understand